src/hosted-buttons/index.js

```diff
@@ -34,24 +41,60 @@ export const getHostedButtonsComponent = (): HostedButtonsComponent => {
         selector,
       });
 
-      // $FlowFixMe
-      Buttons({
+      const createOrder = buildHostedButtonCreateOrder({
+        enableDPoP,
         hostedButtonId,
-        style,
-        onInit,
-        onClick,
-        createOrder: buildHostedButtonCreateOrder({
-          enableDPoP,
-          hostedButtonId,
-          merchantId,
-        }),
-        onApprove: buildHostedButtonOnApprove({
-          enableDPoP,
+        merchantId,
+      });
+      const onApprove = buildHostedButtonOnApprove({
+        enableDPoP,
+        hostedButtonId,
+        merchantId,
+      });
+
+      if (shouldRenderSDKButtons(fundingSources)) {
+        const { flexDirection } = getFlexDirection({ ...style });
+
+        appendButtonContainer({ flexDirection, selector });
+
+        // Only render 2 buttons max
+        fundingSources?.slice(0, 2).forEach((fundingSource, index) => {
+          // $FlowFixMe
+          const standaloneButton = Buttons({
+            hostedButtonId,
+            fundingSource,
+            style: {
+              ...style,
+              color: getButtonColor(style.color, fundingSource),
+            },
+            onInit,
+            onClick,
+            createOrder,
+            onApprove,
+          });
+
+          if (standaloneButton.isEligible()) {
+            standaloneButton.render(
+              index === 0 ? "#ncp-primary-button" : "#ncp-secondary-button"
+            );
+          } else {
+            getLogger().error(`${fundingSource} is not eligible.`);
+          }
+        });
+      } else {
+        // V1 Experience
+        // $FlowFixMe
+        Buttons({
           hostedButtonId,
-          merchantId,
```

Comment:
I think `merchantId` was accidentally removed. And I think it needs to be in the standalone buttons, too for analytics purposes (would be good to double-check why it's needed there, if at all!)

Reply:
sounds good. I added it back to `buttonOptions`